pkg/controller/vitesskeyspace/reconcile_shards.go

```diff
@@ -128,6 +128,7 @@ func newVitessShard(key client.ObjectKey, vtk *planetscalev2.VitessKeyspace, par
 			Namespace: key.Namespace,
 			Name:      key.Name,
 			Labels:    labels,
+			Annotations: shard.VitessShardTemplate.Annotations,
```

Comment:
nit: use the version from the deep copy since we have one already:

```suggestion
			Annotations: template.Annotations,
```

Reply:
Done https://github.com/planetscale/vitess-operator/pull/51/commits/96319b058fcd2e98c87277e0877950b0e885ccf4